sdk/communication/Azure.Communication.CallingServer/src/Models/CallMediaRecognizeDtmfOptions.cs

```diff
@@ -11,16 +11,29 @@ namespace Azure.Communication.CallingServer
     /// </summary>
     public class CallMediaRecognizeDtmfOptions : CallMediaRecognizeOptions
     {
+        private readonly TimeSpan _defaultInterToneTimeout = TimeSpan.FromSeconds(2);
+
         /// <summary> Initializes a new instance of CallMediaRecognizeDtmfOptions. </summary>
         public CallMediaRecognizeDtmfOptions(CommunicationIdentifier targetParticipant) : base(RecognizeInputType.Dtmf, targetParticipant)
         {
+            StopTones = Array.Empty<DtmfTone>();
+            InterToneTimeout = _defaultInterToneTimeout;
         }
 
-        /// <summary> Time to wait between DTMF inputs to stop recognizing. </summary>
+        /// <summary>
+        /// Time to wait between DTMF inputs to stop recognizing.
+        /// If nothing is set, a default of 2 seconds is set.
+        /// </summary>
         public TimeSpan InterToneTimeout { get; set; }
-        /// <summary> Maximum number of DTMFs to be collected. </summary>
+
+        /// <summary>
+        /// Maximum number of DTMFs to be collected.
```

Comment:
DTMF tones to be collected, I think this will be fixed by the updated swagger once i merge my PR in PMA